README.md

```diff
@@ -7,143 +7,33 @@
-
-Use the following command to extract the full set of configuration options into
-a file you can modify:
-
-```console
-$ helm inspect values oci://ghcr.io/brigadecore/brigade-bitbucket-gateway \
-    --version v2.0.0-beta.3 > ~/brigade-bitbucket-gateway-values.yaml
-```
-
-Edit `~/brigade-bitbucket-gateway-values.yaml`, making the following changes:
-
-* `host`: Set this to the host name where you'd like the gateway to be
-  accessible.
-
-* `brigade.apiAddress`: Address of the Brigade API server, beginning with
-  `https://`
-
-* `brigade.apiToken`: Service account token from step 2
-
-* `service.type`: If you plan to enable ingress (advanced), you can leave this
-  as its default -- `ClusterIP`. If you do not plan to enable ingress, you
-  probably will want to change this value to `LoadBalancer`.
-
-Save your changes to `~/brigade-bitbucket-gateway-values.yaml` and use the
-following command to install the gateway using the above customizations:
-
-```console
-$ helm install brigade-bitbucket-gateway \
-    oci://ghcr.io/brigadecore/brigade-bitbucket-gateway \
-    --version v2.0.0-beta.3 \
-    --create-namespace \
-    --namespace brigade-bitbucket-gateway \
-    --values ~/brigade-bitbucket-gateway-values.yaml \
-    --wait \
-    --timeout 300s
-```
-
-### 3. (RECOMMENDED) Create a DNS Entry
-
-If you overrode defaults and set `service.type` to `LoadBalancer`, use this
-command to find the gateway's public IP address:
-
-```console
-$ kubectl get svc brigade-bitbucket-gateway \
-    --namespace brigade-bitbucket-gateway \
-    --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
-```
-
-If you overrode defaults and enabled support for an ingress controller, you
-probably know what you're doing well enough to track down the correct IP without
-our help. 😉
-
-With this public IP in hand, edit your name servers and add an `A` record
-pointing your domain to the public IP.
-
-### 4. Create Webhooks
-
-In your browser, go to your Bitbucket repository for which you'd like to send
-webhooks to this gateway. From the menu on the left, select __Repository
-settings__ and then __Webhooks__. On this page, click __Add webhooks__.
+> ⚠️&nbsp;&nbsp;If you are familiar with the Brigade GitHub Gateway, be advised
+> that this gateway is not as full-featured as that one. At this time, it merely
+> offers parity with its predecessor -- the Brigade v1.x-compatible Bitbucket
+> Gateway -- meaning it only handles simple webhooks and does not offer
+> any integration with
+> [Bitbucket Cloud Apps](https://support.atlassian.com/bitbucket-cloud/docs/bitbucket-cloud-apps-overview/).
+>
+> A more full-featured successor to this gateway is something the Brigade
+> maintainers do wish to explore in collaboration with the Brigade community.
+> Please reach out on the
+> [Kubernetes/#brigade](https://kubernetes.slack.com/messages/C87MF1RFD) Slack
+> channel if you are interested in working on that.
```

Comment:
I think it's best to be clear that this doesn't do everything the GitHub Gateway does, but also be clear that is something that can be tackled if there's enough interest in it -- or better yet -- interest in collaborating on it.